pkg/apis/apps/v1alpha1/description.go

```diff
@@ -59,6 +59,11 @@ type DescriptionSpec struct {
 	//
 	// +optional
 	Raw [][]byte `json:"raw,omitempty"`
+
+	// ChartRaw is the underlying serialization of all helm chart objects.
+	//
+	// +optional
+	ChartRaw [][]byte `json:"chartRaw,omitempty"`
```

Comment:
s/ChartRaw/ChartSpecRaw

Reply:
not only for chart spec but also include chart metadata labels, annotations ...